CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+## [2.0.26] - 2023-03-24
+### Changed
+- @jknipp - Update the gateway options url to use `GET /v1/gateways_options.xml` instead of `OPTIONS /v1/gateways.xml`
 
-## [2.0.24] - 2021-07-12
+## [2.0.25] - 2021-07-12
```

Comment:
@scottmtraver @bradbroge I realized that we did not properly bump the version with the previous update. I've fixed this and will be tagging the current version in `master` as `v2.0.25`.